R/spatial_clustering_cv.R

```diff
@@ -15,14 +15,26 @@
 #'  data are distributed spatially, there may not be an equal number of points
 #'  in each fold.
 #'
+#' You can optionally provide a custom function to `cluster_function`. The
+#' function must take three arguments in a set order: a [stats::dist()] object
+#' indicating distances between data points, a length-1 numeric `v` indicating
+#' the number of folds to create, and `...`. Both the `dist` object and `v`
+#' are passed by position, without names, and must be the first and second
+#' arguments to your function respectively; any named arguments to your
+#' function can be accessed via `...`. The function should return a vector
```

Comment:
Maybe we should pass by name instead and thus require names here? This seems more straightforward to explain and understand.